gemstone/run.sh

```diff
@@ -109,6 +179,18 @@ run_build() {
       ;;
   esac
 
+  # To bypass cached behavior for local build, export GS_TRAVIS_CACHE_ENABLED
+  #   before calling run.sh
+  if [ "${GS_TRAVIS_CACHE_ENABLED}x" = "x" ]; then
```

Comment:
Why is this necessary? One can easily delete the cache by removing the `_cache` directory.
I implemented caching for Squeak and Pharo, because I didn't want to download everything for every build. So that is very useful when I run builds locally to debug a problem.